.circleci/src/@common.yml

```diff
@@ -5,11 +5,15 @@ parameters:
     type: boolean
     description: Whether the pipeline was triggered through the CircleCi API (https://circleci.com/docs/api/v2/?shell#trigger-a-new-pipeline).
     default: false
-  production_delivery:
-    type: boolean
-    description: Whether to deliver to production or not.
-    default: false
-  project:
+  workflow_type:
```

Comment:
We need this to be able to trigger these manually. Otherwise all workflows are triggered. I also don't think it is complicated. If you trigger a workflow, you just specify which workflow you want and everything works fine.

Reply:
Why not just use the solution suggested in the docs? 

`
```
```daily-run-workflow:
  when:
    and:
      - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
```

Reply:
Do you have a link please?

Reply:
https://circleci.com/docs/2.0/scheduled-pipelines/#workflows-filtering